### PR TITLE
API containers ScatterView transition ro rst

### PR DESCRIPTION
**Pull request by marcinwrobel1986**
API/containers/ScatterView transition from .md to .rst

before (.md) | after (.rst)
:----------------: | :-------------:
image |  image
image | ---
image  | image
image | image
--- | image

### CODE REVIEW

File: docs/source/API/containers/ScatterView.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+.. role:: cppkokkos(code)
+	:language: cppkokkos
+
+Header File: ``<Kokkos_ScatterView.hpp>``
+
+.. _parallelReduce: ../core/parallel-dispatch/parallel_reduce.html
+
+.. |parallelReduce| replace:: :cpp:func:`parallel_reduce`
+
```
Comment (ajpowelsnl) [inline docs/source/API/containers/ScatterView.rst:12]: Please make sure the sections are consistently ordered from one API entry to another.  The copy - paste-able code bloc (here, called Usage) should be the last section.  Perhaps rename "Usage" "Example," and make it the last section.  "Description" should come after the `Header File` and should be a brief human language description.  Let's move towards standardizing API entries.

File: docs/source/API/containers/ScatterView.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
+``ScatterView``
+===============
+
+.. role:: cppkokkos(code)
+	:language: cppkokkos
+
+Header File: ``<Kokkos_ScatterView.hpp>``
+
+.. _parallelReduce: ../core/parallel-dispatch/parallel_reduce.html
+
```
Comment (ajpowelsnl) [inline docs/source/API/containers/ScatterView.rst:10]: Why is `parallel_reduce` here?  It's not directly related to ScatterView.

File: docs/source/API/containers/ScatterView.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+       contribute ``ScatterView`` array's results into the input View ``dest``
+
+    .. cppkokkos:function:: reset()
+
+       performs reset on destination array
+
+    .. cppkokkos:function:: reset_except(View<DT, RP...> const& view)
+
+       tbd
```
Comment (ajpowelsnl) [inline docs/source/API/containers/ScatterView.rst:104]: @fnrizzi has some info on `reset_except` that we can use to flesh out syntax and meaning.

File: docs/source/API/containers/ScatterView.rst [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    .. rubric:: *Private* Members
+
+    :member: typedef original_view_type internal_view_type;
+    :member: internal_view_type internal_view;
+
+
+.. rubric:: Free Functions
+
+.. cppkokkos:function:: contribute(View<DT1, VP...>& dest, Kokkos::Experimental::ScatterView<DT2, LY, ES, OP, CT, DP> const& src)
+
```
Comment (ajpowelsnl) [inline docs/source/API/containers/ScatterView.rst:124]: Should the `parallel_reduce` info at the top precede the `contribute` function?

**Review by ajpowelsnl — APPROVED** [review]
A little reorganization is recommended.

File: docs/source/API/containers/ScatterView.rst (hunk 2 above)
Comment (fnrizzi) [reply docs/source/API/containers/ScatterView.rst:10]: it is just a link

File: docs/source/API/containers/ScatterView.rst (hunk 4 above)
Comment (fnrizzi) [reply docs/source/API/containers/ScatterView.rst:124]: no, par_red at the top is just a link

**Review by crtrott — APPROVED** [review]

**Pull request merged**